swo_aws_extension/swo/notifications/teams.py

```
@@ -89,37 +76,91 @@ def send_exception(
-            facts_section.title(facts.title)
-            for key, facts_data in facts.data.items():
-                facts_section.addFact(key, facts_data)
-            message.addSection(facts_section)
-
-        try:
-            message.send()
-        except pymsteams.TeamsWebhookException:
+        """Sends an Adaptive Card to the MS Teams Workflow webhook."""
+        payload = self._build_card(title, text, style, button, facts)
+
+        try:  # noqa: WPS229
```

Ruff (0.15.18)

[warning] Invalid rule code in `# noqa`: WPS229

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)